Library/Homebrew/extend/os/mac/extend/ENV/super.rb

```diff
@@ -94,6 +94,17 @@ def setup_build_environment(formula = nil)
       self["SDKROOT"] = MacOS.sdk_path
     end
 
+    # Filter out symbols known not to be defined on 10.11 since GNU Autotools
+    # can't reliably figure this out with Xcode 8 on its own yet. The Xcode
+    # version doesn't need to be checked since these are harmless with Xcode 7
+    # even if they're unnecessary.
```

Comment:
My concern with that is that it won't fix the problem on a CLT-only install assuming Apple puts the same stuff in the 10.11 Xcode 8 CLT (if there is one actually coming out).